code/binding.go

```diff
@@ -33,7 +33,7 @@ func NewBinding(pkg string, fppath string, fps []*Failpoint) *Binding {
 func (b *Binding) Write(dst io.Writer) error {
 	hdr := "// GENERATED BY GOFAIL. DO NOT EDIT.\n\n" +
 		"package " + b.pkg +
-		"\n\nimport \"github.com/etcd-io/gofail/runtime\"\n\n"
+		"\n\nimport \"github.com/pingcap/gofail/runtime\"\n\n"
```

Comment:
Can we avoid changing import path? It will make it hard for us to merge changes to upstream repo in the future.